test/createBoard.test.js

```diff
@@ -0,0 +1,56 @@
+const request = require('supertest');
+
+const { createApp } = require('../app');
+const { appDataSource } = require('../models/dataSource');
+const jwt = require('jsonwebtoken');
+
+describe('CREATED_BOARD', () => {
+    let app;
+
+    beforeAll(async () => {
+        app = createApp();
+        await appDataSource.initialize();
+    });
+
+    afterAll(async () => {
+        await appDataSource.destroy();
+    })
+
+    test('FAIL : Invalid Access Token Error', async () => {
+        const res = await request(app)
+            .post('/board')
+            .expect({ message : 'INVALID_ACCESS_TOKEN' })
+        expect(res.status).toEqual(401);
+    });
+
+    test('FAIL : User Not Defined Error', async () => {
+        const jwtSpy = jest.spyOn(jwt, 'verify');
+        jwtSpy.mockReturnValue({sub : null});
+        const res = await request(app)
+            .post('/board')
+            .set({ Authorization: 'Some Random Token' })
+            .expect({ message : 'USER_NOT_DEFINED' })
+        expect(res.status).toEqual(404);
+    });
+
+    test('FAIL : KEY Value Error', async() => {
+        const jwtSpy = jest.spyOn(jwt, 'verify');
+        jwtSpy.mockReturnValue({sub : 1});
+        const res = await request(app)
+            .post('/board')
+            .set({ Authorization: 'Some Random Token' })
+            .expect({ message : 'KEY_ERROR' })
+        expect(res.status).toEqual(400);
```

Comment:
```suggestion
        const res = await request(app)
            .post('/board')
            .set({ Authorization: 'Some Random Token' })
            .expect({ message : 'KEY_ERROR' })
            .expect(400)
```

결과를 받아서 expect로 비교할 것인지. 아니면 바로 .expect()를 통해 바로 비교할 것인지 하나로 통일해 주세요.

모든 테스트 코드에 해당하는 코멘트이니 전부 적용해 주세요!

Reply:
.expect() 로 통일하여 반영하였습니다.